src/components/LogRow/CollapsedRow/index.tsx

```diff
@@ -63,21 +59,23 @@ const CollapsedRow: React.FC<CollapsedRowProps> = ({
 
   return (
     <CollapsedLineWrapper data-cy={`collapsed-row-${start}-${end}`}>
-      <StyledOverline>{lineText}</StyledOverline>
-      <StyledButton
-        leftGlyph={<Icon glyph="Expand" />}
-        onClick={expandAll}
-        size="xsmall"
-      >
-        All
-      </StyledButton>
-      <StyledButton
-        leftGlyph={<Icon glyph="Expand" />}
-        onClick={expandFive}
-        size="xsmall"
-      >
-        {SKIP_NUMBER} Above and Below
-      </StyledButton>
+      <StyledBody>{lineText}</StyledBody>
```

Comment:
Looks Great! I wonder if it would be possible to get the background to extend the full width but since this has always been there no need to fix it in this pr.
<img width="660" alt="image" src="https://user-images.githubusercontent.com/4605522/236015809-fe5ecfe7-d051-44e3-bfcc-e18fc43f17ac.png">

Reply:
I'll take a look and see! 😅